libretroshare/src/util/rsdir.cc

```diff
@@ -24,31 +24,31 @@
  *
  */
 
-// Includes for directory creation.
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <fcntl.h>
-#include <unistd.h>
-
-#include "util/rsdir.h"
-#include "util/rsstring.h"
-#include "util/rsrandom.h"
-#include "util/rstime.h"
-#include "util/rsmemory.h"
-#include "util/folderiterator.h"
-#include "retroshare/rstypes.h"
-#include "retroshare/rsnotify.h"
+#include "folderiterator.h"
+#include "rsdir.h"
```

Comment:
why remove the util/ ? it makes the include line more confusing.

Reply:
Done